app/jobs/regular/update_topic_upload_security.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  class UpdateTopicUploadSecurity < ::Jobs::Base
+
+    def execute(args)
+      TopicUploadSecurityManager.new(Topic.find(args[:topic_id])).run
```

Comment:
Do we want this job to throw an exception when the topic can't be found?

Reply:
Good point, probably no need to throw an exception, no need to update upload security in a topic that no longer exists